python/simulation.py

```diff
@@ -1237,7 +1237,9 @@ def modal_volume_in_box(self, box=None, center=None, size=None):
     def solve_cw(self, tol=1e-8, maxiters=10000, L=2):
         if self.fields is None:
             raise RuntimeError('Fields must be initialized before using solve_cw')
-        return self.fields.solve_cw(tol, maxiters, L)
+        res = self.fields.solve_cw(tol, maxiters, L)
+        self._evaluate_dft_objects()
```

Comment:
This line needs to go before `solve_cw`, I think, so that the `update_dfts()` call in `solve_cw` will take effect?

Reply:
Fixed. This is green.